src/Components/Clash/ClashBox/ToppingTwoCard/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import "./ToppingTwoCard.css";
+
+const ToppingTwoCard = (props) => {
```

Comment:
This is pretty darn similar to the ToppingOneCard.

Like I said yesterday I won't make you change this, I'm happy to approve it with the two components, but it would be possible to combine them into a single component if you wanted. Maybe something to look at if you run out of things to do?